### PR TITLE
Readkml: Added right-click menu to map and ability to load to geofence

### PR DESCRIPTION
**Pull request by stephendade**
-Added to the right-click popup menu in the map
-Able to toggle display of individual layers
-Can send polygons to the geofence

### CODE REVIEW

File: MAVProxy/modules/mavproxy_kmlread.py [hunk 1]
```diff
@@ -68,19 +188,49 @@ def loadkml(self, filename):
...
+                self.curtextlayers.append(point[1])                
+
+    def idle_task(self):
+        '''handle GUI elements'''
+        if self.module('map') is not None and not self.menu_added_map:
+            self.menu_added_map = True
+            self.module('map').add_menu(self.menu)
+        #(re)create the menu
+        if mp_util.has_wxpython and self.menu_added_map:
+            # we don't dynamically update these yet due to a wx bug
```
Comment (tridge) [inline MAVProxy/modules/mavproxy_kmlread.py]: I don't think all this needs to be done on every call of idle_task(). That seems to just waste cpu

**Comment by tridge** [comment]
looks very useful! just need to make the idle_task more efficient if we can

**Comment by stephendade** [comment]
@tridge - I've fixed the menu refresh.

**Comment by tridge** [comment]
applied, thanks!

**Pull request closed without merging**